test/match.js

```diff
@@ -8,12 +8,12 @@ describe('match', function() {
 
   it('determines whether a string matches a regex', function() {
     assert.strictEqual(R.match(re, 'B17-afn').length, 1);
-    assert.strictEqual(R.match(re, 'B1-afn'), null);
+    assert.deepEqual(R.match(re, 'B1-afn'), []);
```

Comment:
It would be good to assert that the return value is `[]` rather than `{}`, since `assert.deepEqual` does not distinguish the two. One option is to use `R.toString`:

```javascript
assert.strictEqual(R.toString(R.match(re, 'B1-afn')), '[]');
```